testing/distributions-integ-tests/build.gradle.kts

```diff
@@ -24,9 +24,15 @@ dependencies {
     integTestDistributionRuntimeOnly(projects.distributionsFull)
 }
 
+class LazyString(private val source: Lazy<String>) : java.io.Serializable {
```

Comment:
💭  If we are going to merge this hack, let's add a comment explaining it, and when it should be eligible for removal.
Maybe linking to an existing (underlying) issue that needs to be resolved first?